shared-module/src/components/PeerReview/PeerReviewProgress.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable i18next/no-literal-string */
+import styled from "@emotion/styled"
+import { t } from "i18next"
```

Comment:
Don't import this directly, instead use the useTranslation hook: https://github.com/rage/secret-project-331/blob/master/docs/internationalization.md